examples/02_decoding/plot_haxby_different_estimators.py

```diff
@@ -71,14 +71,18 @@
                       param_grid={'C': [.1, .5, 1., 5., 10., 50., 100.]},
                       scoring='f1', n_jobs=1, cv=3, iid=False)
 
-logistic_cv = GridSearchCV(LogisticRegression(C=1., penalty="l1"),
-                           param_grid={'C': [.1, .5, 1., 5., 10., 50., 100.]},
-                           scoring='f1', cv=3, iid=False)
-logistic_l2_cv = GridSearchCV(LogisticRegression(C=1., penalty="l2"),
-                              param_grid={
-                                  'C': [.1, .5, 1., 5., 10., 50., 100.]
```

Comment:
@kchawla-pi : can you submit a new PR that uses only [.1, 1., 10., 100.] in the two grid search?

It will make things quicker (this example takes 10mn to run), and, with hindsight, it is not necessary to do such a fine grid.

Reply:
Yeah.

Reply:
I see three grid searches and I'll replace them all.